### DISCUSSION TITLE
Feature proposal: comments

### DISCUSSION THREAD

**Discussion by quesada** [zim-desktop-wiki/zim-desktop-wiki#1807] [Ideas]
I tend to write long form in zim and something like this would be awesome: http://criticmarkup.com

It would also help with introducing collaboration into zim, which seems to be a key feature in current times.

**Comment by quesada** [comment]
I would be willing to sponsor this feature

**Comment by quesada** [comment]
Still interested in sponsoring this feature, or anything that starts the path to collaboration, even if async (not looking at gdocs style, two cursors writing at once collab). It's almost 2022 and people are getting used to this type of collaboration

**Reply by introt** [reply]
Hi! I've converted this issue to a discussion for better visibility. Feel free to add [Bounty] to the title!

**Comment by quazardous** [comment]
aside from collaboration, it could be an interesting feature to wrap the whole process of commenting/highlighting a chunk of text. I use this a lot with GoogleDoc.

Commenting (with highlighting) is part of the writing process. So easily adding a comment or a link to a chunk of text could be great.

Maybe it could be merged with the task feature.

The process:

-  select a chunk of text
-  hit CTRL-something or a button
-  enter an optional comment in the confirmation popup (or something else than a popup)
-  a comment/task is created

The rendering could be inline, popinfo or in the task panel only.
